snap/snapcraft.yaml

```diff
@@ -18,13 +18,37 @@ apps:
       - network
       - removable-media
       - home
+  xbcloud:
+    command: usr/bin/xbcloud
+    plugs:
+      - network
```

Comment:
may be add a comment for what reason these plugs are needed? it may help someone new to the repository figure out why access control is set up the way it is